### ISSUE TITLE
Accessibility implementation  in checkbox

### ISSUE THREAD

**Issue by DeepakSap14** [SAP/fundamental-ngx#2602]
#### Is this a bug, enhancement, or feature request?
enhancement

#### Briefly describe your proposal.
1. keyboard Accessibility for checkbox not working.
2. Checkbox without label taking space after checkbox.
<img width="106" alt="Screen Shot 2020-05-26 at 6 48 04 PM (1)" src="https://user-images.githubusercontent.com/57661487/83511762-cdc99a00-a4ec-11ea-8345-c371cfdd2edb.png">

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)
Angular 9 

#### Is there anything else we should know?
Need to take care of keyboard accessibility rules and aria-* tags,

**Comment by salarenko** [comment]
Hey @DeepakSap14 could you be more specific with "keyboard Accessibility for checkbox not working."?

**Comment by DeepakSap14** [comment]
> Hey @DeepakSap14 could you be more specific with "keyboard Accessibility for checkbox not working."?

Hi Salarenko,
checkbox has aria-checked, tabindex, aria-label missing.
Also checkbox check or uncheck does not work on enter key press.

* This issue is created for tracking, my team will raise a PR for this.

**Comment by salarenko** [comment]
Thanks!
You are right about missing aria attributes, but checking checkbox with `Enter` is not a standard behavior and I don't think it should be implemented.

**Comment by JKMarkowski** [comment]
Issue with additional space on empty checkbox is solved here: https://github.com/SAP/fundamental-ngx/pull/2606

**Comment by DeepakSap14** [comment]
It seems Keyboard Accessibility for checkbox includes Enter key check/uncheck of checkboxes.
please refer below link. I am starting development in it.
It will be fix under core/checkbox.

https://wiki.wdf.sap.corp/wiki/display/Acc/ACC-270+Details#ACC-270Details-conformity
https://sapui5.hana.ondemand.com/#/entity/sap.m.CheckBox/sample/sap.m.sample.CheckBox

Please let me know, if anyone has some other thinking.
